### DISCUSSION TITLE
Add a specific playlist for me

### DISCUSSION THREAD

**Discussion by KaydeGRules** [iptv-org/community#1078] [Q&A]
Please add these channels:
KIMT-DT1 (CBS)
KIMT-DT2 (MyNetworkTV)
KIMT-DT3 (Ion Television)
KIMT-DT4 (Antenna TV)
KAAL-DT1 (ABC)
KAAL-DT2 (Start TV)
KAAL-DT3 (Defy TV)
KAAL-DT4 (Bounce TV)
KAAL-DT5 (Scripps News)
KTTC-DT1 (NBC)
KTTC-DT2 (The CW)
KTTC-DT3 (H&I)
KTTC-DT4 (Court TV)
KTTC-DT5 (True Crime Network)
KTTC-DT6 (Outlaw)
KSMQ-DT1 (PBS)
KSMQ-DT2 (Deutsche Welle)
KSMQ-DT3 (Create)
KSMQ-DT4 (Minnesota Channel)
WUMN-LD (Univision)
KYIN-DT1 (PBS)
KYIN-DT2 (PBS Kids)
KYIN-DT3 (World)
KYIN-DT4 (Create)
KDSM-DT2 (Comet)
KDSM-DT3 (Charge!)
KYIN-DT99 (IRIS)
KXSH-LD1 (Telemundo)
KXSH-LD2 (Buzzr)
KXSH-LD3 (QVC)
KXSH-LD4 (TeleXitos)
KXSH-LD5 (Jewelry TV)
KXSH-LD6 (SBN)
KXLT-DT1 (Fox)
KXLT-DT2 (MeTV)
KXLT-DT3 (Laff)
KXLT-DT4 (Ion Mystery)
KXLT-DT5 (Quest)
KXLT-DT6 (Grit)
WDMI-LD1 (Daystar)
WDMI-LD2 (Daystar Espanol)
WDMI-LD3 (Daystar Reflections)

But please do not try and find my locations
This is a request from my mom
If the channels are unavailable right now, try getting them listed as soon as you can

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
